src/Backend/Modules/Location/Js/Location.js

```diff
@@ -19,29 +19,29 @@ export class Location {
     this.width = null
     this.zoomLevel = null
 
+    $('[data-role=toggle-settings]').on('change', function () {
+      console.log('here', $(this))
```

Comment:
forgot to remove log

Reply:
fixed f8da312732f8d3a6276e8f89da6236c219e13433